src/store/modules/app/actions.ts

```diff
@@ -1,6 +1,6 @@
 import { Action } from './types';
```

Comment:
do the filenames need to change to appActions,, etc as well?

Reply:
Do we need that? It's already in the app module right? I can if it would improve your process in some way but it should be easy to find whether you're going through the file navigator or using a search for appActions

Reply:
just wondering, either way i don't mind but im not sure which would be best for maintainability purposes :) can leave as is if you don't think its a problem